### ISSUE TITLE
Problem with conditionals

### ISSUE THREAD

**Issue by kcholoren** [0xJacky/nginx-ui#77]
Hi, I found a problem that I think is new, because this configuration was already working, but yesterday I edited a Site and a problem appeared.
My configuration has several **ifs** inside the _location_ part.
For example:
```
location /XXXX/ {
        proxy_pass YYYY
       ....
        if ($allowed_country = no) {        return 444; }
    }
```
But when I re-edit the site, in the editor it becomes:
```
location /XXXX/ {
        proxy_pass YYYY
       ....
        if ($allowed_country = no);
        return 444;
    }
```
And I get an error when I try to save:
```
Nginx Configuration Parse Error
nginx: [emerg] directive "if" has no opening "{" in /etc/nginx/sites-enabled/tkl-default:83 nginx: configuration file /etc/nginx/nginx.conf test failed 
```
Any ideas why?
Thank you!

**Comment by 0xJacky** [comment]
Thank you for your report. This problem should be fix in 6b9ebbf, and it will be released in next version.

**Comment by kcholoren** [comment]
Thank you!
